### PR TITLE
Fix melt slice variants

### PR DESCRIPTION
**Pull request by ViktorHy**
Disabled filtering of INTRONIC/null/PROMOTER variants from melt-vcf. Instead do intersection with bedtools against designbed. Will only remove unwanted regions and keep relevant.

Before we would miss out on rare splice-site variants.

I also added publishdirs for the unfiltered merged-melt-vcf and for the intersected

### CODE REVIEW

**Comment by Jakob37** [comment]
Nice, I can test run these changes as soon as I have figured out how to run the WGS pipeline (which I am looking into now)

**Comment by alkc** [comment]
Will get started on this later today!

## Review TODO:
- [ ] Review code
- [ ] Start test samples
- [ ] Check if WGS run finishes w/o errors
- [ ] Check if onko run finishes w/o errors
- [ ] Check if onko results can be loaded into scout w/o errors
- [ ] Check if WGS results can be loaded into scout w/o errors

**Comment by ViktorHy** [comment]
I made it a hotfix, should probably be a minor patch. Just realized this now

**Comment by ViktorHy** [comment]
> Nice, I can test run these changes as soon as I have figured out how to run the WGS pipeline (which I am looking into now)

Tell me if you need any help

**Comment by Jakob37** [comment]
> > Nice, I can test run these changes as soon as I have figured out how to run the WGS pipeline (which I am looking into now)
> 
> Tell me if you need any help

Yes, a quick discussion at some point would be nice!

**Comment by Jakob37** [comment]
I got the GIAB dataset to run now. Started a run from this branch as well to compare with on Monday.

**Comment by alkc** [comment]
I'm approving this update

I've test-run a onco v1 and v2 sample and cofnirmed that the pipeline finished w/o errors and that the resulting data can be loaded into scout w/o errors.

I looked at the reported structural variants for both cases and compared them against the respective cases already loaded into scout processed with the live version of the pipeline. No errors found there. The `oncov1` test sample reported one additional SV in an intronic region, which was not found in the live version of the sample results, which I guess is expected given the update.

**Review by alkc — APPROVED** [review]

**Comment by Jakob37** [comment]
I am still fiddling with getting through the onco run. If this is urgent, there is no need to wait for my approval here.

**Review by Jakob37 — APPROVED** [review]
I ran through one onco-sample (with the help of @alkc ). It ran through, and I saw no difference in structural variants.

**Pull request merged**
